orangecontrib/timeseries/widgets/owspiralogram.py

```diff
@@ -11,19 +11,121 @@
+                                 [238, 136, 136], [238, 137, 136],
+                                 [238, 137, 137], [238, 138, 138],
+                                 [238, 139, 139], [239, 140, 140],
+                                 [239, 141, 140], [239, 141, 141],
+                                 [239, 142, 142], [239, 143, 143],
+                                 [240, 144, 144], [240, 145, 144],
+                                 [240, 145, 145], [240, 146, 146],
+                                 [240, 147, 147], [241, 148, 148],
+                                 [241, 149, 148], [241, 149, 149],
+                                 [241, 150, 150], [241, 151, 151],
+                                 [242, 152, 152], [242, 153, 152],
+                                 [242, 153, 153], [242, 154, 154],
+                                 [242, 155, 155], [243, 156, 156],
+                                 [243, 157, 156], [243, 157, 157],
+                                 [243, 158, 158], [243, 159, 159],
+                                 [244, 160, 160], [244, 161, 160],
+                                 [244, 161, 161], [244, 162, 162],
+                                 [244, 163, 163], [245, 164, 164],
+                                 [245, 165, 164], [245, 165, 165],
+                                 [245, 166, 166], [245, 167, 167],
+                                 [246, 168, 168], [246, 169, 168],
+                                 [246, 169, 169], [246, 170, 170],
+                                 [246, 171, 171], [247, 172, 172],
+                                 [247, 173, 172], [247, 173, 173],
+                                 [247, 174, 174], [247, 175, 175],
+                                 [248, 176, 176], [248, 177, 176],
+                                 [248, 177, 177], [248, 178, 178],
+                                 [248, 179, 179], [249, 180, 180],
+                                 [249, 181, 180], [249, 181, 181],
+                                 [249, 182, 182], [249, 183, 183],
+                                 [250, 184, 184], [250, 185, 184],
+                                 [250, 185, 185], [250, 186, 186],
+                                 [250, 187, 187], [251, 188, 188],
+                                 [251, 189, 188], [251, 189, 189],
+                                 [251, 190, 190], [251, 191, 191],
+                                 [252, 192, 192], [252, 193, 192],
+                                 [252, 193, 193], [252, 194, 194],
+                                 [252, 195, 195], [253, 196, 196],
+                                 [253, 197, 196], [253, 197, 197],
+                                 [253, 198, 198], [253, 199, 199],
+                                 [254, 200, 200], [254, 201, 200],
+                                 [254, 201, 201], [254, 202, 202],
+                                 [254, 203, 203], [255, 204, 204]]
+                                )
```

Comment:
OK, if you prefer it this way. For these palettes I personally prefer not having to scroll through them and I rather disable the pylint issue for this line. But this is a personal choice.

Reply:
No, no, no, no! It shouldn't be like that! I explicitly forced PyCharm to keep it in one line several times! What is going on?????

Reply:
I know. Rebasing corrupted it. Now it is fixed.